integration_test/tests/odbc_2_tests.rs

```diff
@@ -105,6 +105,8 @@ mod integration {
     /// call SQLGetTypeInfo to verify the correct types are returned. For all types,
     /// we should get both date types back. For date, we should get the specific date type
     /// we expect back.
+    ///
+    /// NOTE: FLAKY TEST
```

Comment:
As I was testing, I noticed sometimes this test would fail, so I'm guessing it's flaky. Has anyone else noticed this?

Reply:
No I haven't noticed it.  What was the error it was reporting?

Reply:
I see, I looked through the logs and didn't find any other useful error messages to indicate the problem.  I don't see this failing in the waterfall runs so it could be new or specific to your runs.  You can create an investigation ticket to look into it further, although it doesn't have to be high priority unless we see it happen again.

Reply:
Yeah makes sense. Considering it only happened in these instances, I'm guessing it was something to do with my patches (maybe I modified the code in a weird way). Anyways, I'll remove the comment. Thanks for looking into it! I'll keep an eye on it, and if it happens again, I'll make an investigation ticket.